### PR TITLE
Scheduling controlling added

### PR DESCRIPTION
**Pull request by krishrustagi**

### CODE REVIEW

File: emotunes-admin/src/main/java/com/emotunes/emotunes/helper/SchedulingHelper.java [hunk 1]
```diff
@@ -24,17 +26,34 @@ public class SchedulingHelper {
...
+            String modelWeightsUrl = userIdModelWeightsUrlMap.get(userId);
+
+            List<String> songUrlList =
+                    songUrlEmotionList
+                            .stream()
+                            .map(songUrlEmotion -> songUrlEmotion.get(0))
+                            .collect(Collectors.toList());
+
+            List<String> songEmotionList =
+                    songUrlEmotionList
```
Comment (SarthakSavasil) [inline emotunes-admin/src/main/java/com/emotunes/emotunes/helper/SchedulingHelper.java:43]: ye dono list agr dependent hai indexes pr to ye map mai hona chaiye na? ya toh list of dto

File: emotunes-commons/src/main/java/com/emotunes/emotunes/repository/UserSongEmotionPreferenceRepository.java [hunk 2]
```diff
@@ -17,4 +18,11 @@ public interface UserSongEmotionPreferenceRepository extends JpaRepository<Store
                     + " having count(user_id) >= ?1",
             nativeQuery = true)
     List<Tuple> findUserIdSongIdWithUserCountLimit(long limit);
+
+    @Modifying
+    @Query(value =
+            "delete from user_song_emotion_preference"
+                    + " where user_id = ?1",
+            nativeQuery = true)
+    void deleteAllByUserId(String userId);
```
Comment (SarthakSavasil) [inline emotunes-commons/src/main/java/com/emotunes/emotunes/repository/UserSongEmotionPreferenceRepository.java:27]: ye cheez whi hai na jaha agr aage ka process fat jata hai to created entries hatani padengi

Comment (krishrustagi) [reply emotunes-commons/src/main/java/com/emotunes/emotunes/repository/UserSongEmotionPreferenceRepository.java:27]: nhi nhi.. yeh vo hai ki reTraining waala process complete ho jaaye toh preferences mein se vo entries remove krni hai

File: emotunes-admin/src/main/java/com/emotunes/emotunes/helper/SchedulingHelper.java (hunk 1 above)
Comment (krishrustagi) [reply emotunes-admin/src/main/java/com/emotunes/emotunes/helper/SchedulingHelper.java:43]: Yeh sab client calls ke liye pass krna hai... DTO won't work.. don't know flask mein kaise work krta hai

Comment (SarthakSavasil) [reply emotunes-admin/src/main/java/com/emotunes/emotunes/helper/SchedulingHelper.java:43]: accha... map kr sakte hai? But chhod ye abhi shi se map ho rha na.. bss..

File: emotunes-commons/src/main/java/com/emotunes/emotunes/repository/UserSongEmotionPreferenceRepository.java (hunk 2 above)
Comment (SarthakSavasil) [reply emotunes-commons/src/main/java/com/emotunes/emotunes/repository/UserSongEmotionPreferenceRepository.java:27]: accha..

File: emotunes-commons/src/main/java/com/emotunes/emotunes/enums/Emotion.java [hunk 3]
```diff
@@ -1,7 +1,10 @@
 package com.emotunes.emotunes.enums;
 
 public enum Emotion {
+    ANGRY,
+    FEAR,
     HAPPY,
+    NEUTRAL,
```
Comment (SarthakSavasil) [inline emotunes-commons/src/main/java/com/emotunes/emotunes/enums/Emotion.java:7]: ye frontend pe bhi change hoga.. grp pe daal de

**Review by SarthakSavasil — APPROVED** [review]

**Pull request merged**